packages/eslint-config/shared.js

```diff
@@ -20,10 +20,15 @@ module.exports = {
     'plugin:prettier/recommended',
     'plugin:jest-dom/recommended',
   ],
-  parser: 'babel-eslint',
+  parser: '@babel/eslint-parser',
+  parserOptions: {
+    babelOptions: {
+      configFile: '../../babel.config.js',
```

Comment:
Note sure about this one... mind walking me through what this does please?

Reply:
Sure thing, from the OP:
> ### babel-eslint is moved to @babel/eslint-parser
> 
> Overview:
> 
> - https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint
> Migration:
> 
> - https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint
> New parserOptions and adding path to Babel config:
> 
> - https://github.com/babel/babel/tree/main/eslint/babel-eslint-parser#additional-parser-configuration

Parser now needs to know location of the babel.config.js or else it throws ('cause can't find otherwise in our directory structure). There's an option to ignore, but after reading docs that didn't seem correct for our case.

Reply:
Great catch @dac09 I'm now checking cwd and using findUp to get the babel.config, which works across Framework and Project now.

Also, I did confirm we _must_ pass the babel config file to the parser. Breaks without it :-(